Mono.Debugger.Soft/Mono.Debugger.Soft/VirtualMachine.cs

```diff
@@ -613,7 +613,9 @@ class MirrorCache<T> {
 
 		internal EventRequest GetRequest (int id) {
 			lock (requests_lock) {
-				return requests [id];
+				if (requests.ContainsKey(id))
```

Comment:
nit: use `TryGetValue` to avoid double access

Reply:
Done! :)